### ISSUE TITLE
WMI query doesn't always return actual color

### ISSUE THREAD

**Issue by forderud** [forderud/IntelliMouseDriver#32]
The `EvtWmiInstanceQueryInstance` function returns the `TailLightDeviceInformation` context value. The `TailLight` color there is always zero (black) initially, regardless of the actual color.

Probably need to somehow read-back the actual color from the mouse. This is persistent across connect/disconnect, so it cannot be stored in the driver.

## Set color to black when attaching mouse
Need to figure out how to send a SetFeature HIP package from `EvtDeviceAdd`. Some thoughts:
* [WdfDeviceEnqueueRequest](https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/wdfdevice/nf-wdfdevice-wdfdeviceenqueuerequest)
* [WdfRequestCreate](https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/wdfrequest/nf-wdfrequest-wdfrequestcreate), [WdfRequestSend](https://learn.microsoft.com/en-us/windows-hardware/drivers/ddi/wdfrequest/nf-wdfrequest-wdfrequestsend)
* [Using Framework Work Items](https://learn.microsoft.com/en-us/windows-hardware/drivers/wdf/using-framework-work-items)

```
WDF_OBJECT_ATTRIBUTES   attributes;
WDF_OBJECT_ATTRIBUTES_INIT_CONTEXT_TYPE(&attributes, DEVICE_CONTEXT);
attributes.ParentObject = Device;

WDFREQUEST request;
status = WdfRequestCreate(&attributes, MyIoTarget, &request);

// populate request

    // Forward the request down the driver stack
    WDF_REQUEST_SEND_OPTIONS options = {};
    WDF_REQUEST_SEND_OPTIONS_INIT(&options, WDF_REQUEST_SEND_OPTION_SEND_AND_FORGET);

    BOOLEAN ret = WdfRequestSend(Request, WdfDeviceGetIoTarget(device), &options);
    if (ret == FALSE) {
        status = WdfRequestGetStatus(Request);
        KdPrint(("TailLight: WdfRequestSend failed: 0x%x\n", status));
        WdfObjectDelete(Request, status);
    }
```

**Comment by forderud** [comment]
Problem partially solved by 41534e11cf084394a87828252325fd5135cc043d.

**Comment by forderud** [comment]
Problem solved by #39
